ipm_library/ipm_library/ipm.py

```diff
@@ -14,15 +14,14 @@
 
 from typing import Optional
 
-import numpy as np
-import tf2_ros
+from ipm_library import utils
+from ipm_library.exceptions import NoIntersectionError
 from ipm_msgs.msg import PlaneStamped
+import numpy as np
```

Comment:
Is this import formatting standard in ROS2? I find it kind of weird. Is there some guideline for this? In most python projects, they are grouped by `from` and `import`.

Reply:
Ah yes, the import orders were juggled around in  https://github.com/Flova/ipm/pull/1/commits/e38d324180e750b7777e80f58976e12b54ad17e6 to make flake8 happy but I don't really understand why this makes flake8 happy. 

From memory, [flake8: import statements are in the wrong order](https://stackoverflow.com/questions/39739118/flake8-import-statements-are-in-the-wrong-order) was the exact issue that arose and the suggested fix in that link says to add a .flake8 file to configure flake8-import-order plugin is needed.

I couldn't get this working, but juggling around the import orders did fix the issue. Would you know much about the fix suggested on StackOverflow......?

Reply:
Moved conversation to https://github.com/ros-sports/ipm/pull/1#discussion_r830680029